### PR TITLE
Add status bar to bottom of screen during develop

### PR DESCRIPTION
**Pull request by KyleAMathews**
This is a simple start but we'll be adding more things to the
status bar over the next few days.

Screenshot 2019-06-18 17 28 24

### CODE REVIEW

File: packages/gatsby-cli/src/reporter/index.js [hunk 1]
```diff
@@ -179,6 +179,11 @@ const reporter: Reporter = {
       span,
     }
   },
+  _setStage(stage) {
```
Comment (muescha) [inline packages/gatsby-cli/src/reporter/index.js]: why is this method with underscore?

Comment (DSchau) [reply packages/gatsby-cli/src/reporter/index.js]: It's a convention to denote that this method is private and should not be used outside of Gatsby Core.

Comment (muescha) [reply packages/gatsby-cli/src/reporter/index.js]: thx

Comment (KyleAMathews) [reply packages/gatsby-cli/src/reporter/index.js]: yeah, the implementation will change in the near future so I don't want anyone to start using it in the meantime.

File: packages/gatsby/src/commands/develop.js [hunk 2]
```diff
@@ -428,7 +428,14 @@ module.exports = async (program: any) => {
...
   function printInstructions(appName, urls, useYarn) {
-    console.log()
+    report._setStage({
+      stage: `BootstrapFinished`,
+      context: {
+        url: urls.localUrlForBrowser,
+        appName,
+      },
+    })
+    console.log(`\x1B[2J\x1B[0f`)
```
Comment (wardpeet) [inline packages/gatsby/src/commands/develop.js]: what's this? :p maybe add a comment

Comment (KyleAMathews) [reply packages/gatsby/src/commands/develop.js]: clears the screen :-) yeah, good idea — added comment.

**Pull request merged**